lib/asset/AssetGenerator.js

```diff
@@ -144,6 +146,23 @@ class AssetGenerator extends Generator {
 							contentHash
 						}
 					);
+					if (assetModulePublicPath) {
+						if (typeof assetModulePublicPath === "function") {
+							assetModulePublicPath = assetModulePublicPath(
+								filename,
+								module.matchResource || module.resource,
+								runtimeTemplate.compilation.compiler.context
+							);
+						} else {
+							assetModulePublicPath = assetModulePublicPath.endsWith("/")
+								? assetModulePublicPath
+								: `${assetModulePublicPath}/`;
```

Comment:
You should not do it, it is wrong

Reply:
I borrowed the code from this while, https://github.com/webpack-contrib/file-loader/blob/master/src/index.js#L42
🤔 Will there be any problems?

Reply:
It is wrong (we don't change it because we drop `file-loader` in favor `assets` module for webpack v5), publicPath can be any, so we can't add `/` to the end

Reply:
This consideration seems to be right, then I will modify it, thanks for the information